### PR TITLE
LPS-117188 SF

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related issue: https://issues.liferay.com/browse/LPS-117188

### CODE REVIEW

**Comment by carolmariaabb** [comment]
**/cc** @marcosapmf

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/utils/numeric.es.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+	numeral.locale(locale);
+}
+
+setupLocale(Liferay.ThemeDisplay.getLanguageId());
+
+export function formatNumber(number, delimit) {
+	let formattedNumber = number.toString();
+
+	const splittedFormattedNumber = formattedNumber.split('.');
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/utils/numeric.es.js:58]: @marcosapmf, please rename `splittedFormattedNumber` to `formattedNumberParts`. Brian prefers this kind of naming for the outcome of a Split function.

**Comment by marcosapmf** [comment]
@natocesarrego @carolmariaabb the requested changes were made in https://github.com/natocesarrego/liferay-portal/pull/685. Can you guys please close this pull? 

Thanks :)

**Comment by carolmariaabb** [comment]
Closed [#678](https://github.com/natocesarrego/liferay-portal/pull/678).<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**